config/fcos/v1_5_exp/translate.go

```diff
@@ -64,6 +64,13 @@ func (c Config) ToIgn3_4Unvalidated(options common.TranslateOptions) (types.Conf
 		return types.Config{}, translate.TranslationSet{}, r
 	}
 	r.Merge(c.processBootDevice(&ret, &ts, options))
+	for _, disk := range ret.Storage.Disks {
+		for _, partition := range disk.Partitions {
+			if partition.Label != nil && *partition.Label == "root" && partition.Number != 4 {
+				r.AddOnWarn(path.New("json", "fcos", "partition"), common.ErrWrongPartitionNumberForRoot)
```

Comment:
The path isn't arbitrary; it needs to match the actual path from the root of the output to the field.

Reply:
`path.New("json", "storage", "disks", 0, "device", "partitions", 0, "number")` ?

Reply:
More or less, yeah.  `device` isn't in the path, and instead of zeroes, we need to use the indexes of the array entries.